src/Simplex_tree/include/gudhi/Simplex_tree/Simplex_tree_node_explicit_storage.h

```diff
@@ -31,11 +31,14 @@ struct Simplex_tree_node_explicit_storage : SimplexTree::Filtration_simplex_base
   typedef typename SimplexTree::Filtration_value Filtration_value;
   typedef typename SimplexTree::Simplex_key Simplex_key;
 
-  Simplex_tree_node_explicit_storage(Siblings * sib = nullptr,
-                                     Filtration_value filtration = 0)
+  Simplex_tree_node_explicit_storage(Siblings * sib,
+                                     Filtration_value filtration) //MULTIPERS : init to 0 not possible
```

Comment:
Would `Filtration_value filtration = {}` work?

Reply:
I'm not sure it will compile with the standard simplextree (`Filtration_value` is a double)

Reply:
`double d = {};` compiles just fine.